### PR TITLE
refactor: no floating promises

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add eslint rule to force `async` functions to `await`.

Also add a library to do source map for stack trace. Sorry for the lack of PR division.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hal-shu-sato** [comment]
OK. Could you rebase this branch after #711 is merged?
I'll review this PR after that.

File: webpack.renderer.config.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ rules.push({
 plugins.push(new MiniCssExtractPlugin());
 
 module.exports = {
-  devtool: 'source-map',
+  devtool: 'inline-source-map',
```
[the remark this answers is not in the document]
Comment (mitosagi) [reply webpack.renderer.config.js:19]: I was unable to fix the bug that failed to resolve the path of the source map file, so I am unifying it into one file.

File: src/lib/shortcut.ts [hunk 2]
```diff
@@ -35,14 +35,14 @@ export async function removeAviUtlShortcut(appDataPath: string) {
 }
 
 /**
- * Uninstaller for shortcuts. This function MUST be executed before the squirrelCommand is interpreted.
+ * Uninstaller for shortcuts.
```
Comment (mitosagi) [inline src/lib/shortcut.ts:38]: In order for this line to be executed during uninstallation, it must be placed in its original location, someone.

Comment (mitosagi) [reply src/lib/shortcut.ts]: I have resolved it.

**Review by hal-shu-sato — APPROVED** [review]
It works fine!

**Pull request merged**